packages/web-pkg/src/components/CreateLinkModal.vue

```diff
@@ -53,75 +38,82 @@
           : undefined
       "
       :label="passwordEnforced ? `${$gettext('Password')}*` : $gettext('Password')"
-      class="link-modal-password-input oc-mt-l"
+      class="link-modal-password-input"
       :disabled="selectedLinkTypeIsInternal"
       @update:model-value="updatePassword"
     />
+    <div v-else-if="password.value" class="link-modal-password-text oc-text-small oc-text-muted">
+      <span v-text="$gettext('Password: ')" />
```

Comment:
The blank shouldn't be needed because the password is in a new line here in the code.

```suggestion
      <span v-text="$gettext('Password:')" />
```